pkgs/create-neon/data/templates/manifest/scripts.json.hbs

```diff
@@ -6,7 +6,8 @@
   "postcross-build": "neon dist -m /target < cross.log",
   "debug": "npm run cargo-build --",
   "build": "npm run cargo-build -- --release",
-  "cross": "npm run cross-build -- --release"{{#if packageSpec.library}},
+  "cross": "npm run cross-build -- --release",{{#if packageSpec.library}}
   "prepack": "{{#eq packageSpec.library.lang compare="ts"}}tsc && {{/eq}}neon update",
-  "version": "neon bump --binaries platforms && git add ."{{/if}}
+  "version": "neon bump --binaries platforms && git add ."{{else}}
+  "prepare": "npm run build"{{/if}}
```

Comment:
Nit, since this is only for an app, it might be more clear to be `install`.

Although, I'm questioning my suggestion that this is a good idea. There's an odd flow where `package.json` has updated dependencies and we're going to re-build Rust just to install more packages.

Reply:
That's a good point. Given that people can always add the `"install"` or `"prepare"` hook themselves when configuring their app, I think after reflection we should take this back out.